BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -77,20 +76,26 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
         showRestaurants = restaurants
         showStores = stores
         onlyShowOpenStores = openNow
-        distanceFilter = distanceInMiles
+        distanceFilter = DistanceCalculation().milesToMeters(miles: Double(distanceInMiles))
         
+        if let currentLocation = UserLocation.instance.currentCoordinate {
+            
```

Comment:
Too many empty lines going on here.
Please update so that it's just:
```swift
if let currentLocation = ... {
    loadStore(at: currentLocation)
}
```